test/UnitTest/Microsoft.Test.AspNet.OData/Builder/ContainmentPathBuilderTest.cs

```diff
@@ -49,9 +49,9 @@ public class ContainmentPathBuilderTest
         [InlineData("Clubs/ns.SeniorClub(1)/Members(1)/ns.VIP/MyBenefits", "Clubs(1)/Members(1)/ns.VIP")]
         [InlineData("Clubs/ns.SeniorClub(1)/Members/ns.VIP(1)/MyBenefits", "Clubs(1)/Members(1)/ns.VIP")]
         // With redundant EntitySet path
-        [InlineData("Clubs(1)/ns.SeniorClub/Members(1)/ns.VIP/MyAccounts(101)/MyPaymentInstruments", "Clubs(1)/Members(1)/MyAccounts(101)")]
-        [InlineData("Clubs/ns.SeniorClub(1)/Members(1)/ns.VIP/MyAccounts(101)/MyPaymentInstruments", "Clubs(1)/Members(1)/MyAccounts(101)")]
-        [InlineData("Clubs/ns.SeniorClub(1)/Members/ns.VIP(1)/MyAccounts(101)/MyPaymentInstruments", "Clubs(1)/Members(1)/MyAccounts(101)")]
+        [InlineData("Clubs(1)/ns.SeniorClub/Members(1)/ns.VIP/MyAccounts(101)/MyPaymentInstruments", "Accounts(101)")]
```

Comment:
Q: Are we making containedEntitySet directly addressable thru top-level entity-set? But direct access is identified as an issue from this blog https://blogs.msdn.microsoft.com/odatateam/2014/03/12/tutorial-sample-containment-is-coming-with-odata-v4/.

Reply:
No. I think that's the change happened in ODL 7.4.*.